src/fonts/ionicons.scss

```diff
@@ -33,3 +33,20 @@ ion-icon {
   text-transform: none;
   speak: none;
 }
+
+@mixin scale-x($amount) {
```

Comment:
@manucorporat I looked in some code, and it seems you never do custom browser support for `transform`, so I guess I'll do the same. Updated